ee/modules/110-istio/hooks/dataplane_metadata_exporter_test.go

```diff
@@ -126,48 +132,77 @@ var _ = Describe("Istio hooks :: revisions_monitoring ::", func() {
 		})
 	})
 
-	DescribeTable("There are different desired and actual revisions", func(objectsYAMLs []string, want *wantedMetric) {
-		f.ValuesSet("istio.internal.globalVersion", "1.42")
-		yamlState := strings.Join(objectsYAMLs, "\n---\n")
-		f.BindingContexts.Set(f.KubeStateSet(yamlState))
+	DescribeTable("There are different desired and actual revisions",
+		func(objectsYAMLs []string, want *wantedMetric, stats telemetryStatistic) {
```

Comment:
IMO the test case became too bloated. I suggest to make separate test suite for the telemetry to keep the test simpler to read and maintain.

Reply:
ok. i try split it, if it doesn't take long

Reply:
done